examples/device/ep/csrc/vmm.cpp

```
@@ -25,42 +25,151 @@ namespace {
+    cuda_alloc_ctx() : fabric_supported(false), prop({}), granularity(0) {
+        int version;
+
+        if (cuCtxGetDevice(&device) != CUDA_SUCCESS) {
+            throw std::runtime_error("CUDA device should be set before creating a vmm_region");
+        }
+
+        if (cuDriverGetVersion(&version) != CUDA_SUCCESS) {
+            throw std::runtime_error("Failed to get CUDA driver version");
+        }
+
+        if (version < 11000) {
```

Cppcheck (2.20.0)

[style] The function 'get_nvl_buffer_size_hint' is never used.

(unusedFunction)